ДЗ №1

menmanmon opened the pull request.

menmanmon committed:
ДЗ №1

menmanmon committed:
Доделал ДЗ № 1

koretskyhub commented:
Давай вынесем код таймера в отдельный модуль
И сделаем модуль где будет что-то что мы переиспользуем в хотя бы в 2ух других модулях

menmanmon committed:
ДЗ № 1 учел замечания

Вынес код таймера в отдельный модуль, код переключателя переиспользовал в модуле с таймером и в модуле с калькулятором. Плюс код калькулятора тоже вынес в модуль с калькулятором.
